Makefile

```
@@ -1,3 +1,24 @@
+MAIN=dp-reporter-client
+
+BUILD=build
+BUILD_ARCH=$(BUILD)/$(GOOS)-$(GOARCH)
+BIN_DIR?=.
+
+PHONY: all
```

I think several of these phony lines need fixing, thus:
```suggestion
.PHONY: all
```